services/api-server/src/simcore_service_api_server/api/errors/httpx_client_error.py

```diff
@@ -30,6 +31,6 @@ async def handle_httpx_client_exceptions(_: Request, exc: HTTPError):
 
     if status_code >= status.HTTP_500_INTERNAL_SERVER_ERROR:
         _logger.exception("%s. host=%s. %s", detail, exc.request.url.host, f"{exc}")
-    raise HTTPException(
-        status_code=status_code, detail=detail, headers=headers
-    ) from exc
+    return JSONResponse(
```

Comment:
i should have noticed this in the last review ! Sorry! :-)

Reply:
No problem 😄